htdocs/contact/card.php

```diff
@@ -574,7 +574,6 @@
 	$title = $object->lastname;
 }
 $help_url = 'EN:Module_Third_Parties|FR:Module_Tiers|ES:Empresas';
-$title = (getDolGlobalString('SOCIETE_ADDRESSES_MANAGEMENT') ? $langs->trans("NewContact") : $langs->trans("NewContactAddress"));
```

Comment:
Why removing the specialized title used when SOCIETE_ADDRESSES_MANAGEMENT is set ?